### PR TITLE
Move docs from workload fleet

### PR DESCRIPTION
**Pull request by piontec**

### CODE REVIEW

File: docs/update_appcr.md [hunk 1]
```diff
@@ -2,53 +2,100 @@
...
+    ```sh
+    cd management-clusters/${MC_NAME}/organizations/${ORG_NAME}/workload-clusters/${WC_NAME}/apps/${APP_NAME}
+    ```
+
+1. Import the WC's regular GPG private key from your safe storage into your keychain. In our example, we're gonna
+   use `LastPass` for that:
+
+    ```sh
+    gpg --import \
+    <(lpass show --notes "Shared-Dev Common/GPG private key (${MC_NAME}, ${WC_NAME}, Flux)")
```
Comment (ljakimczuk) [inline docs/update_appcr.md:57]: Haven't you mentioned you want to make this docs clean of any `lpass` references?

File: docs/add_mc.md [hunk 2]
```diff
@@ -1,183 +1,189 @@
...
-$ gpg --export-secret-keys --armor "${KEY_FP}" |
-lpass add --notes --non-interactive "Shared-Dev Common/GPG private key (MC_NAME, master, Flux)"
-```
+1. Add the private to a safe encrypted storage of your choice. For example, to export the key to `LastPass`
+   as a secure note, you can run:
 
-5. Delete the private key from the keychain:
+    ```sh
+    gpg --export-secret-keys --armor "${KEY_FP}" |
+    lpass add --notes --non-interactive "Shared-Dev Common/GPG private key (${MC_NAME}, master, Flux)"
```
Comment (ljakimczuk) [inline docs/add_mc.md:65]: Same here

File: docs/add_cluster_crs.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+some that meet your needs, then export their paths with:
+
+```sh
+export CLUSTER_PATH=CLUSTER_BASE_PATH
+export NODEPOOL_PATH=NODEPOOL_BASE_PATH
+```
+
+If desired base is not there, you can add it. Reference the next section to get a general idea of how to do it.
+
+## Create shared template base (optional)
```
Comment (ljakimczuk) [inline docs/add_cluster_crs.md:39]: should we also move template base from our internal repository here? Not sure if we need to do it in this PR, I guess doing this as a followup is also fine

File: docs/update_appcr.md (hunk 1 above)
Comment (piontec) [reply docs/update_appcr.md:57]: I added a comment above that this is just an example of how you can securely store the private key outside of repo. I think this works in this context.

File: docs/add_mc.md (hunk 2 above)
Comment (piontec) [reply docs/add_mc.md:65]: Yeap, as above - there's a comment that this is just an example

File: docs/add_cluster_crs.md (hunk 3 above)
Comment (piontec) [reply docs/add_cluster_crs.md:39]: Yes, definitely, but I thought we make it a separate PR

File: docs/update_appcr.md (hunk 1 above)
Comment (ljakimczuk) [reply docs/update_appcr.md:57]: ah, haven't noticed it, works for me then 👍

**Review by ljakimczuk — APPROVED** [review]

**Comment by ljakimczuk** [comment]
LGTM

**Pull request merged**